packages/app/components/creator-token/buy-creator-token.tsx

```diff
@@ -281,7 +301,7 @@ export const BuyCreatorToken = () => {
                 <Skeleton width={40} height={16} />
               ) : (
                 <Text tw="font-semibold text-gray-700 dark:text-gray-200">
-                  {tokenBalance.data?.toString()}
+                  {tokenBalance.data?.toString() ?? "N/A"}
```

Comment:
Added N/A fallback when no tokenbalance is known

Reply:
yeah, makes sense!